packages/grafana-data/src/transformations/transformers/calculateField.ts

```diff
@@ -230,7 +230,7 @@ function findConsistentTimeFieldName(data: DataFrame[]): string | undefined {
   return name;
 }
 
-export function getResultFieldNameForCalculateFieldTransformerOptions(options: CalculateFieldTransformerOptions) {
+export function getNameFromOptions(options: CalculateFieldTransformerOptions) {
```

Comment:
Since this is no longer global, the name can be reasonable and shared between implementation and the UI